tests/test_tempsensordb.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import os
+from unittest import TestCase, main, skipIf
+from tempDB import TempDB
+
+TEMP_DB = 'temp_tempsensor.db'
+TEMP_TABLE = 'temp_lightclapper'
+PREMADE_DB = 'tests/test_db/test_tempsensor.db'
```

Comment:
Is this file included in the commit?

Reply:
They weren't, part of the final cleanup I can commit those files as well